appconfig/process.csv

```diff
@@ -19,3 +19,4 @@ localhost,{KDBBASEPORT}+16,sortslave,sortslave1,,1,1,,,${KDBCODE}/processes/wdb.
 localhost,{KDBBASEPORT}+17,sortslave,sortslave2,,1,1,,,${KDBCODE}/processes/wdb.q,1,,q
 localhost,{KDBBASEPORT}+18,metrics,metrics1,${TORQHOME}/appconfig/passwords/accesslist.txt,1,1,,,${KDBAPPCODE}/processes/metrics.q,1,,q
 localhost,{KDBBASEPORT}+19,iexfeed,iexfeed1,,1,0,,,${KDBAPPCODE}/processes/iexfeed.q,1,,q
+locathost,{KDBBASEPORT}+20,dqe,dqe1,,1,1,,,${KDBAPPCODE}/processes/dqe.q,1,,q
```

Comment:
typo in hostname

should this process be using `accesslist` to restrict logins?